templates/environment/partials/nat-gateways.yml

```diff
@@ -1,5 +1,7 @@
 {{- range $ind, $cidr := .PrivateSubnetCIDRs}}
 NatGateway{{inc $ind}}Attachment:
+  Metadata:
+    'aws:copilot:description': 'NAT Gateways enabling services placed in private subnets to reach the internet'
```

Comment:
nit: so that it becomes also a possibility for jobs
```suggestion
    'aws:copilot:description': 'NAT Gateways enabling workloads placed in private subnets to reach the internet'
```